src/Illuminate/Http/Client/Pool.php

```diff
@@ -5,14 +5,14 @@
 use GuzzleHttp\Utils;
 
 /**
- * @mixin \Illuminate\Http\Client\Factory
+ * @mixin \Illuminate\Http\Client\PendingRequest
  */
 class Pool
 {
     /**
      * The factory instance.
      *
-     * @var \Illuminate\Http\Client\Factory
+     * @var \Illuminate\Http\Client\PendingRequest
      */
     protected $factory;
```

Comment:
If this is a request, why is the variable $factory?

Reply:
It's actually a PendingRequest template that will be cloned in each pool request. I kept the name since it's like a template for each created request. Lmk if there's a more adequate name.

Reply:
I agree with @taylorotwell, this is highly confusing. We shouldn't change all of this imo in 9.x but in 10.x at the earliest (if we're going to do this).